public/locales/cs/offers.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Accommodation": "Můj barák",
+  "No guests.": "No guests.",
+  "At most one guest.": "Ne víc jak jeden člověk",
+  "At most {{guestNumber}} guests.": "Ne víc jak {{guestNumber}} lidi.",
+  "Can host": "Može mět lidi",
+  "Might be able to host": "Třeba može mět lidi",
+  "Cannot host currently": "Nemože mět lidi",
+  "Hosting status: {{statusLabel}}": "Hosting status: {{statusLabel}}",
+  "I can host": "Možu mět lidi",
```

Comment:
:smile:

Reply:
I find these translations really funny, but shouldn't we have there the actual official czech language? It's likely to land into production eventually.

As a native czech speaker I offer to update this. :)

Reply:
; )